keel-titus-plugin/src/main/kotlin/com/netflix/spinnaker/keel/api/titus/cluster/TitusClusterSpec.kt

```diff
@@ -147,6 +144,16 @@ internal fun TitusClusterSpec.resolveCapacityGroup(region: String) =
 internal fun TitusClusterSpec.resolveConstraints(region: String) =
   overrides[region]?.constraints ?: defaults.constraints ?: Constraints()
 
```

Comment:
This is here because we're updating the spec and not putting the container information in the overrides. I'm not sure of another way that feels less hacky to confirm that this resolved correctly/to get the type right